lms/envs/common.py

```diff
@@ -4151,3 +4151,10 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # COUNTRIES_FIRST = ['SA', 'BH', 'QA'] will display these countries on top of the list
 # https://github.com/SmileyChris/django-countries#show-certain-countries-first
 COUNTRIES_FIRST = []
+
+################# Settings for brand logos. #################
```

Comment:
Where are the actual default values for these logos set?  

I.e., the "Default Out-of-the-box config values" described here: https://openedx.atlassian.net/wiki/spaces/AC/pages/2047345474/Logo+Flip+Plan+and+Instructions#Logo-related-Configuration-Settings

Why is it that the email logo has a different usage pattern/constant naming scheme?  For ace_common/template_context.py below, it appears we use `DEFAULT_EMAIL_LOGO_URL` if `LOGO_URL_PNG` isn't present.  Do we need a similar pattern for other usages of the logos?  Or should the values here in common.py actually hard code the defaults?

Reply:
I can answer these questions. 

1. Here are the default values defined for prod edx ([edx-internal](https://github.com/edx/edx-internal/blob/master/edx-remote-config/prod/lms.yml#L1063-L1068)). Similarly, there are defined in edge-internal for edge.edx.org. 

2. Email logo URL is a little different in a way 1) it should be a web URL in this case (`LOGO_URL_PNG`), but if that has not been defined, we are using the default openedX logo. 

3. I think the right step forward should be to get this value from edx-internal like other logo urls to keep it consistent. 

4. I think there is room to improve the variable name `DEFAULT_EMAIL_LOGO_URL` I am open to suggestions.

Reply:
As of now, I believe we can go ahead and merge this PR and come back and further refine as we go. The state of our settings is not great in our platform and there's a lot of room for improvement.

Reply:
Happy to move forward as @nasthagiri suggests - just want to clarify that my question is more about Open edX defaults, since non-edx.org usage of this won't use edx-internal.